neuvector-nstools-5.3.yaml

```
@@ -0,0 +1,38 @@
+package:
+  name: neuvector-nstools-5.3
+  version: 5.3.2
+  epoch: 0
+  description: "NeuVector NSTools"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: edbdcba632835d56dcc92ba86323c8a196471289
+      repository: https://github.com/neuvector/neuvector.git
+      tag: v${{package.version}}
+
+  - runs: |
+      make -C tools/nstools
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
+      install -Dm755 tools/nstools/nstools ${{targets.contextdir}}/usr/local/bin/nstools
+
+update:
+  enabled: true
+  github:
+    identifier: neuvector/neuvector
+    strip-prefix: v
+    tag-filter: v5.3.
+
+test:
+  pipeline:
+    - name: Run Help Command
+      runs: /usr/local/bin/nstools || true
```

```suggestion
      runs: /usr/local/bin/nstools |grep 'h: help'
```

I think this will work well enough. If `grep` doesn't find that string it will exit with a non-zero status.